paddle/fluid/pir/dialect/operator/ir/ops.yaml

```
@@ -968,6 +968,18 @@
     func : prod
   backward : prod_grad
 
+- op : pull_sparse_v2
+  args : (Tensor[] ids, Tensor[] w, int embeddingdim = 11, int tableid = 0, str accessorclass = "", str ctrlabelname = "", int paddingid = 0, bool scalesparsegrad = true, str[] inputnames = {}, bool is_distributed = true)
+  output : Tensor[](out)
```

```suggestion
  output : Tensor[](out){ids.size()}
```
另一个问题应该是这里要明确 out的大小

---

收到